weave/trace_server_bindings/remote_http_trace_server.py

```diff
@@ -188,19 +139,33 @@ def _flush_calls(
             self._flush_calls(batch[split_idx:], _should_update_batch_size=False)
             return
 
-        # Send the batch to the server with retries
-        self._send_batch_to_server(encoded_data)
-
-    @tenacity.retry(
-        stop=tenacity.stop_after_delay(REMOTE_REQUEST_RETRY_DURATION),
-        wait=tenacity.wait_exponential_jitter(
-            initial=1, max=REMOTE_REQUEST_RETRY_MAX_INTERVAL
-        ),
-        retry=tenacity.retry_if_exception(_is_retryable_exception),
-        before_sleep=_log_retry,
-        retry_error_callback=_log_failure,
-        reraise=True,
-    )
+        # If a single item is too large, we can't send it - raise an error
+        if encoded_bytes > self.remote_request_bytes_limit and len(batch) == 1:
+            raise ValueError(
```

Comment:
what happens if we raise? should this just be a logger.error and pass?

Reply:
Yeah we can do that